include/zephyr/pm/state.h

```diff
@@ -281,6 +295,60 @@ struct pm_state_info {
 			Z_PM_STATE_INFO_FROM_DT_CPU, (), node_id)	       \
 	}
 
+/**
+ * @brief Initialize an array of struct pm_state_info with information from all
+ * the states present and enabled in the given device node identifier.
+ *
+ * Example devicetree fragment:
+ *
+ * @code{.dts}
+	soc {
+ *		...
+ *		cpu0: cpu@0 {
+ *			device_type = "cpu";
+ *			...
+ *			cpu-power-states = <&state0 &state1>;
+ *		};
+
+ *		i2c1: i2c@50000000 {
+ *			...
+ *			reinit-power-states = <&state1>;
+ *		};
+ *
+ *		power-states {
```

Comment:
cpus/power-states do not belong to the soc (simple-bus) node, use `cpus`.

Reply:
Good catch, will update the docs.